tests/reports/test_report_views.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from django.contrib.contenttypes.models import ContentType
+from django.core import mail
+from django.core.urlresolvers import reverse
+from rest_framework import status
+
+
+@pytest.mark.django_db
+def test_anonymous_user_can_not_report(apiclient):
+    url = reverse('reports-list')
+    data = {}
+    response = apiclient.post(url, data, format='json')
+    assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+@pytest.mark.django_db
+def test_anonymous_user_can_not_view_reportlist(apiclient):
+    url = reverse('reports-list')
+    response = apiclient.get(url, format='json')
+    assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+@pytest.mark.django_db
+def test_authenticated_user_can_report(apiclient, user):
```

Comment:
Maybe I'm overlooking something, but IMO the test name does not match the behaviour.